integration/tests/basic/erc/test_ERC721.py

```diff
@@ -113,14 +112,18 @@ def test_symbol(self, erc721):
         assert symbol == "MPL"
 
     def test_balanceOf(self, erc721):
-        balance_before = erc721.contract.functions.balanceOf(erc721.account.address).call()
+        with allure.step("Get balance for user"):
```

Comment:
this additional allure.step doesn't provide additional information without context. If will a problem we will understand where without this steps.

Reply:
Following the task goal: "Add more debug information for big cases (erc721 for example, add manually additional info) in the context of allure" - this step adds information to allure report test body, and we can see now in test body that first we got balance, second we did mint, and lastly we check balance again. Before that test body in allure report was just empty.

However as asked, I removed them.

Reply:
Yes, more debugging. But just comment and step without additional info (for example balances, tx, user id) is useless. Just remove and make assert more clear (but you already)